src/Loadbalancers/Entities/Freetype.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+namespace UKFast\SDK\Loadbalancers\Entities;
+
+use UKFast\SDK\Entity;
+
+/**
+ * @property string $id
+ * @property string $uuid
```

Comment:
```suggestion
 * @property string $aclId
```

These are the properties on the loadbalancer API
```
* @property mixed id
 * @property mixed string
 * @property mixed acl_id
```